crates/mun_codegen/src/apple.rs

```diff
@@ -1,4 +1,6 @@
-use once_cell::sync::OnceCell;
+use once_cell::sync::Lazy;
+use std::collections::HashMap;
+use std::sync::Mutex;
```

Comment:
Use `parking_lot::Mutex`. It's guaranteed to succeed

Reply:
Nice fixed!